plugins/vpp/ifplugin/interface_state.go

```diff
@@ -22,46 +22,46 @@ import (
 	"sync"
 	"time"
 
+	"git.fd.io/govpp.git/adapter"
+
 	govppapi "git.fd.io/govpp.git/api"
 	"github.com/go-errors/errors"
 	"github.com/ligato/cn-infra/logging"
 	"github.com/ligato/cn-infra/utils/safeclose"
 	"github.com/ligato/vpp-agent/plugins/govppmux"
 	"github.com/ligato/vpp-agent/plugins/vpp/binapi/interfaces"
-	"github.com/ligato/vpp-agent/plugins/vpp/binapi/stats"
 	"github.com/ligato/vpp-agent/plugins/vpp/ifplugin/ifaceidx"
 	intf "github.com/ligato/vpp-agent/plugins/vpp/model/interfaces"
 )
 
-// counterType is the basic counter type - contains only packet statistics.
-type counterType int
+// PeriodicPollingTimeout between statistics reads
+// TODO set to the original VPP interval but should be configurable
+var PeriodicPollingTimeout = 10 * time.Second
```

Comment:
The _timeout_ term is not correct here. The variable defines interval/period and not timeout.

Reply:
Also, please set the default to 1 second. The polling period does not really affect performance in new stats API, because of the optimistic concurrency algorithm it uses which does not interrupt main worker.